internal/component/loki/source/docker/runner.go

```diff
@@ -52,6 +52,9 @@ type options struct {
 
 	// positions interface so tailers can save/restore offsets in log files.
 	positions positions.Positions
+
+	// taskRestartInterval to restart task that has stopped running.
+	taskRestartInterval time.Duration
```

Comment:
Sorry, missed this last time. This causes a periodical call to `t.target.StartIfNotRunning()`, so I think it should be called `targetRestartInterval` instead.